bcipy/simulator/task/copy_phrase.py

```diff
@@ -18,11 +18,27 @@
 from bcipy.task.paradigm.rsvp.copy_phrase import RSVPCopyPhraseTask
 
 DEFAULT_EVIDENCE_TYPE = EvidenceType.ERP
+SUPPORTED_EVIDENCE_TYPES = {EvidenceType.ERP.value}
 
 
 def get_evidence_type(model: SignalModel) -> EvidenceType:
-    """Get the evidence type provided by the given model"""
-    return model.metadata.evidence_type or DEFAULT_EVIDENCE_TYPE
+    """Get the evidence type provided by the given model and return it as an enum (EvidenceType).
+
+    If the model does not provide an evidence type, the default evidence type is returned.
+    """
+    evidence_type = model.metadata.evidence_type
+
+    # Note: once more than one evidence type is supported, this should be updated to throw an error
+    if not evidence_type:
+        return DEFAULT_EVIDENCE_TYPE
+
+    # Check if the evidence type is supported
+    if evidence_type not in SUPPORTED_EVIDENCE_TYPES:
+        raise ValueError(f"Unsupported evidence type: {evidence_type}. Supported types: {SUPPORTED_EVIDENCE_TYPES}")
```

Comment:
I would prefer that the task doesn't make the call as to what evidence is supported. When we add the Gaze support we shouldn't need to change anything in the task. Maybe just try to create the EvidenceType from the string and let that fail if it's not valid.